lib/desktop/cli.rb

```diff
@@ -249,6 +249,18 @@ def cli_syntax(command, args_str = nil)
 EOF
     end
 
+    command :resize do |c|
+      cli_syntax(c, 'ID WIDTHxHEIGHT')
+      c.summary = 'Resize an existing desktop session'
+      c.action Commands, :resize
+      c.description = <<EOF
+Resize an existing desktop session.
+
+Specify the desktop ID of the session and the desired size in the format 
+WIDTHxHEIGHT.
+EOF
+      end
```

Comment:
Can you add a paragraph explaining how to specify the desktop.  We have such a paragraph for our other commands such as `show` and `rename`.  We should be consistent here too.